### DISCUSSION TITLE
How to make my wallpaper known?

### DISCUSSION THREAD

**Discussion by iArisu** [rocksdanister/lively#520] [Q&A]
Hello,

> I wrote a WallPaper which uses HTML5 canvas and javascript.
I find it original, I think it is aimed to address young audiences.
Could you tell me by which channel to make it known?

Thank you, looking forward to read you

**Comment by rocksdanister** [comment]
You can post it in the subreddit:
https://www.reddit.com/r/LivelyWallpaper/

If you can make it a video preview post it will get peoples attention better.
(You can create video capture within Lively itself rightclick wallpaper-> Preview -> Start.)
